--- pt-fingerprint.orig	2019-09-23 20:21:51.000000000 +0900
+++ pt-fingerprint	2019-09-23 20:22:55.000000000 +0900
@@ -37,7 +37,7 @@
 
 use List::Util qw(max);
 use Getopt::Long;
-use Data::Dumper;
+#use Data::Dumper;
 
 my $POD_link_re = '[LC]<"?([^">]+)"?>';
 
@@ -1102,7 +1102,7 @@
       %default_vars, # first the tool's defaults
       %user_vars,    # then the user's which overwrite the defaults
    );
-   PTDEBUG && _d('--set-vars:', Dumper(\%vars));
+   #PTDEBUG && _d('--set-vars:', Dumper(\%vars));
    return \%vars;
 }
 
@@ -1935,8 +1935,8 @@
 package pt_fingerprint;
 
 use English qw(-no_match_vars);
-use Data::Dumper;
-$Data::Dumper::Indent = 1;
+#use Data::Dumper;
+#$Data::Dumper::Indent = 1;
 $OUTPUT_AUTOFLUSH     = 1;
 
 use constant PTDEBUG => $ENV{PTDEBUG} || 0;
